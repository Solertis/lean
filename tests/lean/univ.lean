--

definition id2 (A : Type*) (a : A) := a

check id2 Type* num

check id2 Type* num


check id2 Type num

check id2 _ num

check id2 (Sort (_+1)) num

check id2 (Sort (0+1)) num

check id2 Type* (Type 1)

check id2 (Type*) (Type 1)
