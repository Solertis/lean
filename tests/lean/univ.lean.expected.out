id2 Type num : Type
id2 Type num : Type
id2 Type num : Type
id2 Type num : Type
id2 Type num : Type
id2 Type num : Type
id2 Type 2 Type 1 : Type 2
id2 Type 2 Type 1 : Type 2
